NEFI_functions/extract_ndep.r

```diff
@@ -0,0 +1,48 @@
+#This function takes longitude and latitude (in that order) as inputs.
+#returns a dataframe of 2000-2015 averages on wet N deposition, dry deposition, and total N deposition.
+#There are currently warnings. ignore them. 
+
+extract_ndep <- function(longitude,latitude){
+  #load dry deposition rasters
+  dry.list <- list()
+  for(i in 0:15){
+    dry.list[[i+1]] <- raster(paste0('/fs/data3/caverill/CASTNET_Ndep/dry_dep/n_dw-',2000 + i,'.e00'))
```

Comment:
I'd recommend against hardcoding file paths within function definitions. Better would be to pass an argument to the function, e.g.
```
extract_ndep <- function(longitude,latitude,folder='/fs/data3/caverill/CASTNET_Ndep/dry_dep/'){
```
and then build up the file name from that folder
```
dry.list[[i+1]] <- raster(file.path(folder,paste0('n_dw-',2000 + i,'.e00')))
```